services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/attachment/application/ApplicationAttachmentService.kt

```diff
@@ -63,6 +63,10 @@ class ApplicationAttachmentService(
         attachmentType: ApplicationAttachmentType,
         attachment: MultipartFile
     ): ApplicationAttachmentMetadata {
+        logger.info {
+            "Adding attachment to application, applicationId = $applicationId, " +
+                "attachment name = ${attachment.originalFilename}, size = ${attachment.bytes.size}"
```

Comment:
Content type could be added also. Feel free to ignore.

Reply:
That's a good idea, added it as well.